src/server/replica.cc

```diff
@@ -913,8 +914,17 @@ void DflyShardReplica::StableSyncDflyReadFb(ExecutionState* cntx) {
       force_ping_ = true;
       journal_rec_executed_.fetch_add(1, std::memory_order_relaxed);
     } else {
-      ExecuteTx(std::move(*tx_data), cntx);
-      journal_rec_executed_.fetch_add(1, std::memory_order_relaxed);
+      auto exe_res = ExecuteTx(std::move(*tx_data), cntx);
```

Comment:
please do not use `auto` for POD types or one word types - it reduces the readability